CHANGELOG.md

```diff
@@ -1,3 +1,350 @@
+cd $ASSETS_DIR/terraform
+export WORKER_POOL=""
+export WORKER_COUNT=""
+
+for ((i=0;i<$WORKER_COUNT;i++)); do
+  terraform state rm "module.worker-$WORKER_POOL.packet_bgp_session.bgp[$i]"
+done
+cd -
+```
+
+Repeat above steps for each worker pool running MetalLB with BGP enabled on it.
+
+2. Install Packet CCM
+
+  Export your Packet cluster's project ID and API Key.
+
+  ```bash
+  export PACKET_API_KEY=""
+  export PACKET_PROJECT_ID=""
+
+  echo "apiKey: $PACKET_API_KEY
+projectID: $PACKET_PROJECT_ID" > /tmp/ccm-values.yaml
+
+  # TODO: Update this to 0.6.0
+  # curl -LO https://github.com/kinvolk/lokomotive/archive/v0.5.0.tar.gz
+  # tar -xvzf v0.5.0.tar.gz
+  # helm install packet-ccm --namespace kube-system --values=/tmp/ccm-values.yaml lokomotive-0.5.0/assets/charts/control-plane/packet-ccm/
+
+  # TODO: Remove this before merging
+  curl -LO https://github.com/kinvolk/lokomotive/archive/master.zip
+  unzip master.zip
+  helm install packet-ccm --namespace kube-system --values=/tmp/ccm-values.yaml lokomotive-master/assets/charts/control-plane/packet-ccm/
+  ```
+
+3. Add permanent labels to the nodes running MetalLB, just run the following command and label will be added:
+
+Create the script which will be used to modify the env var file. So even if the node is restarted these labels will persist:
+
+```bash
+echo '
+#!/bin/bash
+
+set -euo pipefail
+
+echo -e "Updating Kubelet env file...\nOld Kubelet env file:\n"
+cat /etc/kubernetes/kubelet.env
+
+label=$(grep ^NODE_LABELS /etc/kubernetes/kubelet.env)
+label_prefix="${label::-1}"
+augmented_label="${label_prefix},lokomotive.alpha.kinvolk.io/bgp-enabled=true\""
+
+# This copy is needed because sed tries to create a new file. And this changes inode and docker does not allow it.
+cp /etc/kubernetes/kubelet.env /kubelet.env
+sed -i "s|^NODE_LABELS.*|${augmented_label}|g" /kubelet.env
+cat /kubelet.env > /etc/kubernetes/kubelet.env
+
+echo -e "\nNew Kubelet env file:\n"
+cat /etc/kubernetes/kubelet.env
+
+echo -e "\nUpdating Kubelet config creator file...\nOld Kubelet config creator file:\n"
+cat /etc/kubernetes/configure-kubelet-cgroup-driver
+
+cp /etc/kubernetes/configure-kubelet-cgroup-driver /configure-kubelet-cgroup-driver
+sed -i "s|^EOF|providerID: external|g" /configure-kubelet-cgroup-driver
+echo "EOF" >> /configure-kubelet-cgroup-driver
+cat /configure-kubelet-cgroup-driver > /etc/kubernetes/configure-kubelet-cgroup-driver
+
+echo -e "\nNew Kubelet config creator file:\n"
+cat /etc/kubernetes/configure-kubelet-cgroup-driver
+
+echo -e "\nRestarting Kubelet...\n"
+exec nsenter -a -t 1 /bin/sh -c "systemctl restart kubelet && systemctl status --no-pager kubelet"
+' | tee /tmp/run
+```
+
+Store above script in a configmap in a separate namespace:
+
+```bash
+kubectl create ns update-host-files
+kubectl create -n update-host-files cm script --from-file /tmp/run --dry-run=client -o yaml | kubectl apply -f -
+```
+
+Run this command to create pods on each node that is running metallb:
```

Comment:
- Does the user needs to select which nodes to run the commands on? If not, let's drop the extra explanation and say "Run the following command".
- MetalLB should be capitalized correctly.